storage/pkg/flagset/storagehome.go

```diff
@@ -23,95 +23,121 @@ func StorageHomeWithConfig(cfg *config.Config) []cli.Flag {
 		// Storage home
 
 		&cli.StringFlag{
-			Name:        "network",
+			Name:        "grpc-network",
 			Value:       "tcp",
 			Usage:       "Network to use for the storage service, can be 'tcp', 'udp' or 'unix'",
-			EnvVars:     []string{"STORAGE_STORAGE_HOME_NETWORK"},
-			Destination: &cfg.Reva.StorageHome.Network,
-		},
-		&cli.StringFlag{
-			Name:        "protocol",
-			Value:       "grpc",
-			Usage:       "protocol for storage service, can be 'http' or 'grpc'",
-			EnvVars:     []string{"STORAGE_STORAGE_HOME_PROTOCOL"},
-			Destination: &cfg.Reva.StorageHome.Protocol,
+			EnvVars:     []string{"STORAGE_HOME_GRPC_NETWORK"},
+			Destination: &cfg.Reva.StorageHome.GRPCNetwork,
 		},
 		&cli.StringFlag{
-			Name:        "addr",
+			Name:        "grpc-addr",
 			Value:       "0.0.0.0:9154",
 			Usage:       "Address to bind storage service",
-			EnvVars:     []string{"STORAGE_STORAGE_HOME_ADDR"},
-			Destination: &cfg.Reva.StorageHome.Addr,
+			EnvVars:     []string{"STORAGE_HOME_GRPC_ADDR"},
+			Destination: &cfg.Reva.StorageHome.GRPCAddr,
 		},
 		&cli.StringFlag{
-			Name:        "url",
-			Value:       "localhost:9154",
-			Usage:       "URL to use for the storage service",
-			EnvVars:     []string{"STORAGE_STORAGE_HOME_URL"},
-			Destination: &cfg.Reva.StorageHome.URL,
+			Name:        "http-network",
+			Value:       "tcp",
+			Usage:       "Network to use for the storage service, can be 'tcp', 'udp' or 'unix'",
+			EnvVars:     []string{"STORAGE_HOME_HTTP_NETWORK"},
+			Destination: &cfg.Reva.StorageHome.HTTPNetwork,
 		},
-		&cli.StringSliceFlag{
-			Name:    "service",
-			Value:   cli.NewStringSlice("storageprovider"),
-			Usage:   "--service storageprovider [--service otherservice]",
-			EnvVars: []string{"STORAGE_STORAGE_HOME_SERVICES"},
+		&cli.StringFlag{
+			Name:        "http-addr",
+			Value:       "0.0.0.0:9155",
+			Usage:       "Address to bind storage service",
+			EnvVars:     []string{"STORAGE_HOME_HTTP_ADDR"},
+			Destination: &cfg.Reva.StorageHome.HTTPAddr,
 		},
 
+		// TODO allow disabling grpc / http services
+		/*
+			&cli.StringSliceFlag{
+				Name:    "grpc-service",
+				Value:   cli.NewStringSlice("storageprovider"),
+				Usage:   "--service storageprovider [--service otherservice]",
+				EnvVars: []string{"STORAGE_HOME_GRPC_SERVICES"},
+			},
+			&cli.StringSliceFlag{
+				Name:    "http-service",
+				Value:   cli.NewStringSlice("dataprovider"),
+				Usage:   "--service dataprovider [--service otherservice]",
+				EnvVars: []string{"STORAGE_HOME_HTTP_SERVICES"},
+			},
+		*/
+
 		&cli.StringFlag{
 			Name:        "driver",
-			Value:       "owncloud",
+			Value:       "ocis",
 			Usage:       "storage driver for home mount: eg. local, eos, owncloud, ocis or s3",
-			EnvVars:     []string{"STORAGE_STORAGE_HOME_DRIVER"},
+			EnvVars:     []string{"STORAGE_HOME_DRIVER"},
```

Comment:
need to be changed in drone https://github.com/owncloud/ocis/blob/d422a91a01b98893d0a10a7555092342755ee33b/.drone.star#L546

Reply:
Done.